### PR TITLE
Add Level selection buttons component

### PR DESCRIPTION
**Pull request by Ars6300**

### CODE REVIEW

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,16 +1,14 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```
Comment (nattallius) [inline src/app/app.component.spec.ts:1]: If you want to use CUSTOM_ELEMENTS_SCHEMA, you should place it in app.module, not in .spec file)
Just curious, why do you need it now?)

File: src/app/app.module.ts [hunk 2]
```diff
@@ -1,21 +1,16 @@
+import { SelectorsComponent } from './modules/level-select/components/selectors/selectors.component';
+import { SelectLevelComponent } from './modules/level-select/level-select.component';
+import { LeftLayoutComponent } from './shared/components/left-layout/left-layout.component';
```
Comment (nattallius) [inline src/app/app.module.ts]: Useless import)

File: src/app/modules/level-select/components/selectors/selectors.component.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="selectors">
+  <h1 class="selectors-title">{{ title }}</h1>
+
+  <div class="selectors-container">
+    <button class="selectors-container_button" *ngFor="let button of buttons">
+      {{ button | uppercase }}
+    </button>
+    <button class="selectors-container_button">Start Test</button>
```
Comment (nattallius) [inline src/app/modules/level-select/components/selectors/selectors.component.html]: What is the reason to use underscore before button? why just simple leave selectors-container-button?

**Review by nattallius — COMMENTED** [review]
Despite my comments, I saw you used Interface and ngFor directive, cool work!

File: src/app/app.component.spec.ts (hunk 1 above)
Comment (Ars6300) [reply src/app/app.component.spec.ts:1]: I had error saying that I should use it😅

File: src/app/modules/level-select/components/selectors/selectors.component.html (hunk 3 above)
Comment (Ars6300) [reply src/app/modules/level-select/components/selectors/selectors.component.html]: isn't it BEM style?))

File: src/app/app.component.spec.ts (hunk 1 above)
Comment (nattallius) [reply src/app/app.component.spec.ts:1]: To be honest, I also had to use it only after an error occurred)

**Review by nattallius — APPROVED** [review]

**Review by DobrovskayaElena — APPROVED** [review]

**Pull request merged**